nixos/modules/services/misc/mbpfan.nix

```diff
@@ -29,23 +29,23 @@ in {
 
     maxFanSpeed = mkOption {
       type = types.int;
-      default = 6200;
+      default = 6199;
       description = ''
         The maximum fan speed.
       '';
     };
 
     lowTemp = mkOption {
       type = types.int;
-      default = 63;
+      default = 55;
```

Comment:
I'm not sure it's wise to deviate from upstream's defaults. It's easy enough to configure as a user anyway?

Reply:
Re-reading this. Maybe upstream hasn't provided sensible defaults for aarch64-darwin yet? That'd be sufficient reason to deviate from the defaults, imo. Would require testing on a variety of cpus ideally. And maybe it should only deviate if the platform is aarch64-darwin?

Reply:
I have no idea about M1. I have Macbook Pro from 2014 and that has the issue. I will refactor the module so I am closing this for now.